### DISCUSSION TITLE
configure Nextflow to execute jobs in existing pool

### DISCUSSION THREAD

**Discussion by gabimirea** [nextflow-io/nextflow#3569] [General]
Hello,
How I should specify /  config Nextflow to execute jobs in an existion pool?
I don't find any example how to use this directive "queue"
Does this need to be added in nextflow.config or in the nf file? 
I am using an azure batch and azure storage account with a private vnet.
I configured the pool in azure interface with private vnet.

**Comment by colindaven** [comment]
Add to `nextflow.config` in the executor section, heres an example for lsf.
```

executor {
  name = 'lsf'
  //name = 'local'
  perJobMemLimit = true
  queue = 'hpcqueue'
  memory = '40 GB'
  cpus = 16

}

```
